src/tests/integration-test

```diff
@@ -246,7 +251,8 @@ class UDisksTestCase(unittest.TestCase):
         If cd is True, return the CD device, otherwise the hard disk device.
         """
         block = klass.udisks_block(partition, cd)
-        return klass.client.get_object(block.get_object_path()).get_filesystem()
+        return klass.client.get_object(
+            block.get_object_path()).get_filesystem()
```

Comment:
> If the code was passing clean with these tools, it would be easy to check for future violations when adding/changing new code. At the moment things are so non-conforming that anything new will just be lost in the noise.

You can always tell pep8 to ignore some errors using `--ignore`. We run pep8 with `--ignore=E501,E402,E731` in [blivet](https://github.com/rhinstaller/blivet/blob/2.1-devel/Makefile#L80) and other python projects.